docs/pipelines.md

```diff
@@ -282,6 +284,28 @@ In this example, the task "build-the-image" will be executed and if the first
 run fails a second one would triggered. But, if that fails no more would
 triggered: a max of two executions.
 
+
+#### conditions
+
+Sometimes you will need to run tasks only when some conditions are true. The `conditions` field 
+allows you to list a series of references to [`Conditions`](#conditions) that are run before the task
```

Comment:
i think you might want `(conditions.md)` instead of `(#conditions)` here? (to link to the conditions doc vs linking back to the section we're already in)

Reply:
done